PKHUD/FrameView.swift

```diff
@@ -14,20 +14,24 @@ internal class FrameView: UIVisualEffectView {
 
     internal init() {
         super.init(effect: UIBlurEffect(style: .light))
-        commonInit()
+        DispatchQueue.main.async {
+            self.commonInit()
+        }
     }
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
-        commonInit()
+        DispatchQueue.main.async {
+            self.commonInit()
+        }
     }
 
-    fileprivate func commonInit() {
+    private func commonInit() {
         backgroundColor = UIColor(white: 0.8, alpha: 0.36)
```

Comment:
Why don't you put DispatchQueue.main.async {...} here instead of outside in the init constructors above?